src/vm/hints/hint_utils.rs

```diff
@@ -280,6 +280,81 @@ pub fn assert_le_felt(
     }
 }
 
+//Implements hint:from starkware.cairo.common.math_cmp import is_le_felt
+//    memory[ap] = 0 if (ids.a % PRIME) <= (ids.b % PRIME) else 1
+pub fn is_le_felt(
+    vm: &mut VirtualMachine,
+    ids: HashMap<String, BigInt>,
+) -> Result<(), VirtualMachineError> {
+    //Check that ids contains the reference id for each variable used by the hint
+    let (a_ref, b_ref) = if let (Some(a_ref), Some(b_ref)) =
+        (ids.get(&String::from("a")), ids.get(&String::from("b")))
+    {
+        (a_ref, b_ref)
+    } else {
+        return Err(VirtualMachineError::IncorrectIds(
+            vec![String::from("a"), String::from("b")],
+            ids.into_keys().collect(),
+        ));
+    };
+    //Check that each reference id corresponds to a value in the reference manager
+    let (a_addr, b_addr) = if let (Some(a_addr), Some(b_addr)) = (
+        get_address_from_reference(a_ref, &vm.references, &vm.run_context),
+        get_address_from_reference(b_ref, &vm.references, &vm.run_context),
+    ) {
+        (a_addr, b_addr)
+    } else {
+        return Err(VirtualMachineError::FailedToGetIds);
+    };
+    match (vm.memory.get(&a_addr), vm.memory.get(&b_addr)) {
+        (Ok(Some(maybe_rel_a)), Ok(Some(maybe_rel_b))) => {
+            for (name, builtin) in &vm.builtin_runners {
+                //Check that range_check_builtin is present
+                if name == &String::from("range_check") {
```

Comment:
```suggestion
                if name != &String::from("range_check") {
                    continue;
                }
```
Then reduce an indentation level of everything inside.

Reply:
Alternatively we could break with the answer and work outside of the loop when found.